### PR TITLE
Implemented only google sign in

### PR DESCRIPTION
**Pull request by ksdrof500**

### CODE REVIEW

**Review by neteinstein — APPROVED** [review]

File: core/src/commonMain/kotlin/com/mindera/people/UiState.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.mindera.people
+
+sealed class UiState<out T> {
```
Comment (magnumrocha) [inline core/src/commonMain/kotlin/com/mindera/people/UiState.kt:3]: It's better to move this to `utils` package and rename it to `DataState`.

File: android/src/main/java/com/mindera/people/android/ui/home/HomeViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    private fun setUser(email: String?, name: String?) {
+        User(email = email, name = name).run {
+            _userState.value = UiState.Success(this)
+            authViewModel.authenticate(this)
+        }
+    }
+
+    private fun checkSignedInUser() {
+        GoogleSignIn.getLastSignedInAccount(context)?.run {
```
Comment (magnumrocha) [inline android/src/main/java/com/mindera/people/android/ui/home/HomeViewModel.kt:39]: I don't think the ViewModel is the place to put this kind of code.
I would prefer to make this on the View part and pass the Result to ViewModel, maybe we can create a Composable to hold it and use this as base wrapping the Screens inside it.

File: android/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -3,4 +3,7 @@
 
     <string name="loading">Loading&#8230;</string>
     <string name="retry">Retry</string>
+    <string name="auth_error_msg">Something went wrong...</string>
+    <string name="google_cloud_server_client_id" translatable="false">856677742822-vdq7toqtgkdhb4jfguokcua701cp2uio.apps.googleusercontent.com</string>
```
Comment (magnumrocha) [inline android/src/main/res/values/strings.xml:7]: just to clarify: Is this a public id or a private id? is it ok this to be committed to the repository?

File: android/src/main/java/com/mindera/people/android/ui/home/HomeViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import com.mindera.people.auth.AuthViewModel
+import com.mindera.people.user.User
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+class HomeViewModel(
+    private val context: Context,
+    private val authViewModel: AuthViewModel
+) : ViewModel() {
```
Comment (magnumrocha) [inline android/src/main/java/com/mindera/people/android/ui/home/HomeViewModel.kt:17]: use the ViewModel definition of the `core` module: `com.mindera.people.utils.ViewModel`

**Review by magnumrocha — COMMENTED** [review]
Please check my comments.

File: android/src/main/res/values/strings.xml (hunk 3 above)
Comment (magnumrocha) [reply android/src/main/res/values/strings.xml]: As we talked about and researched this, the Client ID token is public.
So, no Issue here.

**Review by magnumrocha — APPROVED** [review]

**Pull request merged**